### ISSUE TITLE
BezelProject not showing in RetroPie menu.

### ISSUE THREAD

**Issue by Samington33** [thebezelproject/BezelProject#83]
Hi, Just wondering whether it’s safe and/or recommended to run the “bezelproject.sh” shell script directly from Terminal?

The thing is, after running the commands listed in the readme, there is nothing related to the Bezel Project in my RetroPie menu. I have verified that the file is present; I navigated to “/home/pi/RetroPie/retropiemenu” through the Terminal, and ran “ls”. “bezelproject.sh” was listed, alongside the expected options — “Bluetooth.rp” “WiFi.rp” etc.

I’m storing everything — Operating System, ROMs — on an SSD, as opposed to a Micro-SD, as I have the RetroFlag NESPi4 case. Could this be an issue?

Apologies if I’m missing something obvious; I’m really new to Linux.

Any help appreciated.

**Comment by thebezelproject** [comment]
Yes, you can run the bezelproject.sh script manually from a terminal. Hard to say at this point why it doesn't show in the menu system. First try running script from a terminal. Select a themed bezel set for a set of roms you already have installed, then run the retroarch entry for that system and see if the bezels show up.

**Comment by dragon57** [comment]
That error usually means you need to perform a full update of the OS and RetroPie. Have you tried that?

**Comment by Samington33** [comment]
Okay, so this is all sorted now.

Someone on the RetroPie forum recommended that I disable the “Parse Gamelists Only” option, and this did cause the “Bezel Project” to appear in the menu. After I launched and immediately exited it, I was able to re-enable “Parse Gamelists Only”. Hopefully this can help others who might be experiencing the same issue.

As regards the “System Error: Unexpected”, it doesn’t seem to be happening anymore. Previously, I was running the script with sh “bezelproject.sh”. As per another recommendation on the RetroPie forum, I tried ./bezelproject.sh. This caused the script to run properly. Of course, as the menu option is now working, there’s no longer any need to run the script from Terminal.

I have been using the RetroPie Setup Script to keep installed packages up-to-date.

Thanks for all the help.
